torch/_inductor/ir.py

```diff
@@ -2072,6 +2083,9 @@ def get_size(self):
     def get_stride(self):
         return list(self.layout.stride)
 
+    def get_workspace_size(self):
+        return 0
+
```

Comment:
Should we move this to `TemplateBuffer`, as that seems to be the lowest common subclass where this is relevant? Or is it necessary for arbitrary `Buffer` instances to be able to return this?

Reply:
I feel it depends on the algorithm. In theory triton codegen code can also require extra workspace.

Reply:
Also, I think, you're invoking this method on abstract `Buffer` in some context, I remember. So ignore the comment.